jujugui/static/gui/src/app/components/machine-view/test-machine-view.js

```diff
@@ -320,12 +329,13 @@ describe('MachineView', function() {
         addGhostAndEcsUnits={sinon.stub()}
         autoPlaceUnits={autoPlaceUnits}
         createMachine={createMachine}
+        destroyMachines={sinon.stub()}
         environmentName="My Env"
-        units={units}
```

Comment:
Thanks for alphabetizing these; as the list of props gets more unwieldy it'll be nice to have that in place.